.github/workflows/nightly.yml

```diff
@@ -103,93 +102,250 @@ jobs:
         timeout-minutes: 60
         run: python3 test/functional/test_runner.py --combinedlogslen=4000
 
-  windows:
-    runs-on: windows-latest
-    env:
-      PYTHONUTF8: 1
-      QT_DOWNLOAD_URL: 'https://github.com/sipsorcery/qt_win_binary/releases/download/v1.6/Qt5.9.8_x64_static_vs2019.zip'
-      QT_DOWNLOAD_HASH: '9a8c6eb20967873785057fdcd329a657c7f922b0af08c5fde105cc597dd37e21'
-      QT_LOCAL_PATH: 'C:\Qt5.9.8_x64_static_vs2019'
-      VCPKG_INSTALL_PATH: "$env:VCPKG_INSTALLATION_ROOT/installed"
-      PLATFORM: x64
+  linux (ASAN+UBSAN):
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - name: checkout
+        uses: actions/checkout@v1
+        with:
+          path: main
       - name: Extract branch name & commit
         id: extract_branch
         run: |
-          $env:git_hash = git.exe rev-parse --short=7 "$env:GITHUB_SHA"
-          $env:branch = echo $env:GITHUB_REF#refs/heads/ | %{$_ -replace '\/','.'}
-      - uses: actions/setup-python@v1
+          echo "::set-output name=git_hash::$(git rev-parse --short=7 "$GITHUB_SHA")"
+          echo "::set-output name=branch::$(echo ${GITHUB_REF#refs/heads/} | sed 's/\//\./g')"
+          echo "::set-output name=commits_ahead::$(git rev-list $(git describe --tags $(git rev-list --tags --max-count=1)).. --count)"
+          echo "::set-output name=last_tag::$(git describe --tags $(git rev-list --tags --max-count=1))"
+      - name: get version for veriblock-pop-cpp
+        run: echo "::set-env name=VERIBLOCK_POP_CPP_VERSION::$(awk -F '=' '/\$\(package\)_version/{print $NF}' $PWD/depends/packages/veriblock-pop-cpp.mk | head -n1)"
+      - name: veriblock-pop-cpp release version
+        run: echo Using veriblock-pop-cpp version $VERIBLOCK_POP_CPP_VERSION
+      - name: add bitcoin ppa
+        run: sudo add-apt-repository ppa:bitcoin/bitcoin
+      - name: install build requirements
+        run: sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils python3
+      - name: install dependencies
+        run: sudo apt-get install libevent-dev libboost-system-dev libboost-filesystem-dev libboost-test-dev libboost-thread-dev libminiupnpc-dev libzmq3-dev libdb4.8-dev libdb4.8++-dev libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libqrencode-dev libubsan0 libubsan1 libtsan0 libasan4
+      - name: install sonar
+        env:
+          SONAR_CLI_VERSION: 4.2.0.1873
+        run: |
+          (
+          set -e; \
+          mkdir -p /opt/sonar; \
+          curl -L -o /tmp/sonar.zip https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-${SONAR_CLI_VERSION}-linux.zip; \
+          unzip -o -d /tmp/sonar-scanner /tmp/sonar.zip; \
+          mv /tmp/sonar-scanner/sonar-scanner-${SONAR_CLI_VERSION}-linux /opt/sonar/scanner; \
+          sudo ln -s -f /opt/sonar/scanner/bin/sonar-scanner /usr/local/bin/sonar-scanner; \
+          rm -rf /tmp/sonar*
+          )
```

Comment:
we don't need sonar in this build

Reply:
ah, missed that.  It was removed from running, but not from installing, will push a fix shortly.